### PR TITLE
[modular] updates the SAD messages, and removes the EMP act

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

Changes the messages to no longer reference a real company (and one from some cartoon i've forgotten, it's a reference of some sort I've forgotten), and a families gang, and instead the generic medical company appended to other items, while also making the repeating messages less 'funny', 

Also, removes the emp-act, because being put into the monkey doesn't actually transfer your mind, thus, RR'ing you and I honestly doubt there's be a justifiable reason to emag the SAD anyway.

if @hugmeorelse  has an in-lore company that'd work better, violently DM me and i'll change it

TL:DR Less LRP 'lubeman' messages, removes a company that was ham fisted into the setting, and else wise, makes the character-restoring device less ahelp-bait, by removing the sudden RR after emagging it.

## Changelog

:cl:
code: changed both the in-desc, and in message makers of the SAD, it's looping messages during use, and removed it's mostly broken emag act
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 1]
```diff
@@ -35,12 +35,11 @@
...
-	"Thank you for using the Self-Actualization Device, brought to you by Cinco: A Family Company!", \
+	"Thank you for using the Self-Actualization Device, brought to you by Veymed, because you asked for it.", \
 	"The Self-Actualization device is not to be used by the elderly without direct adult supervision. Cinco is not liable for any and all injuries sustained under unsupervised usage of the Self-Actualization Device.", \
 	"Please make sure to clean the Self-Actualization Device every fifteen minutes! The Self-Actualization Device is not to be used un-cleaned.", \
-	"Before using the Self-Actualization Device, remove your teeth. You don't want your pearly whites getting in the way of the Self-Actualization Device!" , \
-	"Please make sure to have your pre-Self Actualization Device brain-stimulating full-body gel lube session performed by a licensed lube man.", \
-	"Have more questions about the Self-Actualization Device? Call your Cinco Mancierge to requisition more information about the Self-Actualization Device!" \
+	"Before using the Self-Actualization Device, remove any and all metal devices, or you might make the term 'ironman' a bit to literal!" , \
```
Comment (tf-4) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm]:
```suggestion
	"Before using the Self-Actualization Device, remove any and all metal devices, or you might make the term 'ironman' a bit too literal!" , \
```

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 2]
```diff
@@ -14,12 +14,12 @@
 
 /obj/machinery/self_actualization_device
 	name = "Self-Actualization Device"
-	desc = "With the power of modern neurological scanning and synthflesh cosmetic surgery, the Morningstar corporation \
-	has teamed up with the Cinco Chemical & Toy Division to bring you the Self-Actualization Device! \
+	desc = "With the power of modern neurological scanning and synthflesh cosmetic surgery, the Veymed(ical) Corporation \
+	has teamed up with Nanotrasen Human (And elsewise) resources to bring you the Self-Actualization Device! \
```
Comment (tf-4) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm]:
```suggestion
	has teamed up with Nanotrasen Human (and elsewise) resources to bring you the Self-Actualization Device! \
```

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 3]
```diff
@@ -35,12 +35,11 @@
 	var/next_fact = 10
 	/// A list containing advertisements that the machine says while working.
 	var/static/list/advertisements = list(\
-	"Thank you for using the Self-Actualization Device, brought to you by Cinco: A Family Company!", \
+	"Thank you for using the Self-Actualization Device, brought to you by Veymed, because you asked for it.", \
 	"The Self-Actualization device is not to be used by the elderly without direct adult supervision. Cinco is not liable for any and all injuries sustained under unsupervised usage of the Self-Actualization Device.", \
 	"Please make sure to clean the Self-Actualization Device every fifteen minutes! The Self-Actualization Device is not to be used un-cleaned.", \
-	"Before using the Self-Actualization Device, remove your teeth. You don't want your pearly whites getting in the way of the Self-Actualization Device!" , \
-	"Please make sure to have your pre-Self Actualization Device brain-stimulating full-body gel lube session performed by a licensed lube man.", \
```
Comment (tf-4) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:42]: I kind of like the lube man line idk

Comment (RatFromTheJungle) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:42]: It was removed to make all the lines vaguely around the same tier, and because a machine telling you to get your full-body lubing from a lubeman isn't exactly a good look if we're trying to aim for 'more serious.'

Comment (tf-4) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:42]: good enough

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 4]
```diff
@@ -14,12 +14,12 @@
 
 /obj/machinery/self_actualization_device
 	name = "Self-Actualization Device"
-	desc = "With the power of modern neurological scanning and synthflesh cosmetic surgery, the Morningstar corporation \
-	has teamed up with the Cinco Chemical & Toy Division to bring you the Self-Actualization Device! \
+	desc = "With the power of modern neurological scanning and synthflesh cosmetic surgery, the Veymed(ical) Corporation \
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm]: Why the `(ical)` here? That looks a bit odd.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm (hunk 4 above)
Comment (RimiNosha) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm]: Yea, in lore, the company is known as VeyMed, and is only VeyMedical Corporation in official documents, according to current lore document on the company's parent empire (non-public for now iirc).

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 5]
```diff
@@ -14,12 +14,12 @@
...
+	has teamed up with Nanotrasen Human (and elsewise) resources to bring you the Self-Actualization Device! \
```
Comment (RimiNosha) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm]: I'd say swap `(and elsewhere)` and `resources` around so it reads better.

```diff
+ has teamed up with Nanotrasen Human Resources (and elsewhere) to bring you the Self-Actualization Device!

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 6]
```diff
@@ -157,18 +156,14 @@
 	var/brute_damage = patient.getBruteLoss()
 	var/burn_damage = patient.getFireLoss()
 
-	if(obj_flags & EMAGGED)
-		patient.monkeyize()
```
Comment (Zonespace27) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:161]: Tested locally: does not ghost people

Comment (tf-4) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:161]: Seen it ghost people on live

Comment (RatFromTheJungle) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:161]: > Tested locally: does not ghost people

it ghosted me when I tested it in the tdome

i'd append it to the same thing that causes mindswap to crash clients

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me.
